WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -518,6 +521,19 @@ private extension CardPresentPaymentsOnboardingUseCase {
     }
 }
 
+// MARK: - Analytics
+private extension CardPresentPaymentsOnboardingUseCase {
+    func trackPluginInstallFailed(_ error: Error) {
+        guard let countryCode = self.storeCountryCode else {
+            DDLogError("Error installing plugin: \(error)")
+            return
+        }
+        ServiceLocator.analytics.track(event: .InPersonPayments.cardPresentOnboardingCtaFailed(reason: "plugin_install_tapped",
```

Comment:
Since there's only one call to analytics I opted to use a ServiceLocator instance for now. I might change this to inject it into the initializer if we add more event tracking to the class. Happy to change it with this PR if you think is appropriate though 👍